xt/40-dbsetup.t

```diff
@@ -16,9 +16,7 @@ HINT:  Set LSMB_NEW_DB environment variable and try running again.');
 
 my $temp = $ENV{TEMP} || '/tmp/';
 my $run_tests = 1;
```

Comment:
I think this variable can be eliminated too.

Reply:
Agreed... to follow... one step at a time...